### PR TITLE
Fixed highest cp

### PR DESCRIPTION
**Pull request by pvanassen**
The thing gave back EEVEEs on evolved EEVEEs

### CODE REVIEW

File: library/src/main/java/com/pokegoapi/api/pokemon/PokemonDetails.java [hunk 1]
```diff
@@ -305,7 +305,10 @@ public int getMaxCpFullEvolveAndPowerupForPlayer() throws LoginFailedException,
 	 */
 	private int getMaxCpFullEvolveAndPowerup(int playerLevel) {
 		PokemonIdOuterClass.PokemonId highestUpgradedFamily;
-		if (getPokemonId() == EEVEE) {
+		if (asList(VAPOREON, JOLTEON, FLAREON).contains(getPokemonId())) {
+			highestUpgradedFamily = getPokemonId();
+		}
+		else if (getPokemonId() == EEVEE) {
```
Comment (FabianTerhorst) [inline library/src/main/java/com/pokegoapi/api/pokemon/PokemonDetails.java:311]: } and else if has to be in One line when you want to follow the code Style.

Comment (pvanassen) [inline library/src/main/java/com/pokegoapi/api/pokemon/PokemonDetails.java:311]: I prefer this style thanks

Comment (FabianTerhorst) [inline library/src/main/java/com/pokegoapi/api/pokemon/PokemonDetails.java:311]: Are you sure checkstyle runs without errors. Last time with this it prints an error

Comment (pvanassen) [inline library/src/main/java/com/pokegoapi/api/pokemon/PokemonDetails.java:311]: Same here but I'm out of time to fix it

**Comment by FabianTerhorst** [comment]
Can be merged when checkstyle is fixed.

**Comment by FabianTerhorst** [comment]
checkstyle wasn´t fixed

**Comment by AbandonedCart** [comment]
Problem solved. Now it is no longer being held up by a minor line edit.

**Pull request merged**